### PR TITLE
Updated the Get saved zones endpoint

### LINKED ISSUE #48: S.2 End point GET: Send the stored maps
The goal of this task is to store the saved zones in the database. To do so the frontend team needs to send the information via POST method.

According to the defined data structures. Implement the POST (and other if needed) function to store the data in the MongoDB.

The task is considered to be done if by pressing a button save the zones are stored in the MongoDB.

### PR DESCRIPTION
**Pull request by Patricijia**

### CODE REVIEW

**Review by VictorVader — DISMISSED** [review]

File: backend/cmd/web/bzone_plot_handler.go [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+	//set query filter
+	queryFilter := bson.D{{Key: "plot_id", Value: reqBZonePlot}}
+
+	var bZonePlot models.PlotModel
+	err = plotsCollection.FindOne(context.TODO(), queryFilter).Decode(&bZonePlot)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusInternalServerError)
+		return
+	}
```
Comment (NeaGogu) [inline backend/cmd/web/bzone_plot_handler.go]: move database stuff to /intern/models in a separate function (look how I did it for zipcodes). That way the functionality can be tested individually.

**Review by NeaGogu — CHANGES_REQUESTED** [review]

File: backend/cmd/web/bzone_plot_handler.go (hunk 1 above)
Comment (Patricijia) [reply backend/cmd/web/bzone_plot_handler.go]: @NeaGogu the entire GetBZonePlot?

Comment (Patricijia) [reply backend/cmd/web/bzone_plot_handler.go]: Because I dont have subqueries

Comment (Patricijia) [reply backend/cmd/web/bzone_plot_handler.go]: yea i think i understood

Comment (NeaGogu) [reply backend/cmd/web/bzone_plot_handler.go]: just the selected code (lines 20 - 31)

**Comment by NeaGogu** [comment]
and fix the conflicts pls

**Review by VictorVader — DISMISSED** [review]
Watch out for the conflicts!

**Review by VictorVader — APPROVED** [review]

**Review by NeaGogu — APPROVED** [review]

**Pull request merged**